components/cli/internal/test/file_system.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package test
+
+import (
+	"io/ioutil"
+)
+
+type mockFileSystem struct {
+}
+
+// CurrentDir returns the current directory.
+func (fs *mockFileSystem) CurrentDir() (string, error) {
+	return ioutil.TempDir("", "cellery-current-dir")
+
+}
+
+// UserHome returns the user home.
+func (fs *mockFileSystem) UserHome() string {
+	userHome, _ := ioutil.TempDir("", "user-home")
+	return userHome
```

Comment:
what happens if an error happens while creating?

Reply:
This is to match UserHome method signature of FileSystem interface. However updated UserHome interface function to return string and error.